phone_verify/backends/base.py

```diff
@@ -56,29 +64,51 @@ def token_expired(cls, stored_verification):
 
     def create_security_code_and_session_token(self, number):
         """
-        Creates a temporary security_code inside the cache, this holds the phone number
-        as value, so that we can later check if everything is correct.
+        Creates a temporary ``security_code`` and ``session_token`` inside the DB.
 
-        It also generates a session_code for the client to send in subsequent requests.
+        ``security_code`` is the code that user would enter to verify their phone_number.
+        ``session_token`` is used to verify if the subsequent call for verification is
+        by the same device that initiated a phone number verification in the
+        first place.
 
-        :param number: Number of recipient
+        :param number: Phone number of recipient
 
         :return security_code: string of sha security_code
-        :return session_code: string of session_code
+        :return session_token: string of session_token
         """
-        security_code = self.generate_token()
-        session_code = self.generate_session_token(security_code, number)
+        security_code = self.generate_security_code()
+        session_token = self.generate_session_token(security_code, number)
 
         # Delete old security_code(s) for phone_number if already exists
         SMSVerification.objects.filter(phone_number=number).delete()
 
         # Default security_code generated of 6 digits
         SMSVerification.objects.create(
-            phone_number=number, security_code=security_code, session_code=session_code
+            phone_number=number,
+            security_code=security_code,
+            session_token=session_token,
         )
-        return security_code, session_code
+        return security_code, session_token
 
-    def validate_token(self, security_code, phone_number, session_code):
+    def validate_security_code(self, security_code, phone_number, session_token):
+        """
+        A utility method to verify if the ``security_code`` entered is valid for
+        a given ``phone_number`` along with the ``session_token`` used.
+
+        :param security_code: Security code entered for verification
+        :param phone_number: Phone number to be verified
+        :param session_token: Session token to identify the device
+
+        :return stored_verification: Contains the verification object
+        corresponding to the phone_number if found, else None.
+        :return status: Status for the stored_verification object.
+        Can be one of the following:
+            - `BaseBackend.SECURITY_CODE_VALID`
+            - `BaseBackend.SECURITY_CODE_INVALID`
+            - `BaseBackend.SECURITY_CODE_EXPIRED`
+            - `BaseBackend.SECURITY_CODE_VERIFIED`
+            - `BaseBackend.SESSION_TOKEN_INVALID`
```

Comment:
Need to enclose above status codes in double backticks as well.

Reply:
This only helps within the rST format, single backquotes are okay for comments

Reply:
Check out https://www.python.org/dev/peps/pep-0287/#id61

It states:

`single backquotes is recognized as "interpreted text", whose interpretation is application-dependent`

`double-backquotes to indicate program I/O or code snippets.`